app/src/main/res/menu/home_menu.xml

```diff
@@ -50,6 +50,10 @@
       android:title="@string/menu_jump"
       app:showAsAction="never" />
   <item
+      android:id="@+id/translations"
+      android:title="Translations"
```

Comment:
ah ok, i misunderstood that ignore my bottom comment. I can add it in there. jzk